### PR TITLE
fix: fix missing introduction and image in app template

### PR DESCRIPTION
**Pull request by ROOMrepair**
### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes

Links #https://github.com/kubesphere/project/issues/4860

### Special notes for reviewers

```

```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc

```docs

```

### CODE REVIEW

**Review by donniean — APPROVED** [review]

File: packages/shared/src/components/Apps/AppInformation/index.tsx [hunk 1]
```diff
@@ -3,29 +3,42 @@
...
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Card } from '@kubed/components';
 import { useParams } from 'react-router-dom';
 
 import { useCacheStore as useStore } from '../../../index';
 import { AppInfo, LabelText } from './AppInfo';
 import type { AppDetail } from '../../../types';
-import { useAppVersionList } from '../../../stores/openpitrix';
+// import { useAppVersionList } from '../../../stores/openpitrix';
```
Comment (Leioy) [inline packages/shared/src/components/Apps/AppInformation/index.tsx:13]: 没有用到的直接删了吧，不用注释。

**Review by donniean — APPROVED** [review]

**Review by Leioy — APPROVED** [review]

**Pull request merged**